scripts/get_predictoor_info.py

```diff
@@ -1,253 +1,8 @@
-
-
-def date_to_unix(date_string):
-    dt = datetime.strptime(date_string, "%Y-%m-%d")
-    return int(time.mktime(dt.timetuple()))
-
-
-def get_statistics(all_predictions):
-    total_predictions = len(all_predictions)
-    correct_predictions = 0
-
-    stats = {"pair_timeframe": {}, "predictoor": {}}
-
-    for prediction in all_predictions:
-        pair_timeframe_key = (prediction.pair, prediction.timeframe)
-        predictoor_key = prediction.user
-        source = prediction.source
-
-        is_correct = prediction.prediction == prediction.trueval
-
-        if pair_timeframe_key not in stats["pair_timeframe"]:
-            stats["pair_timeframe"][pair_timeframe_key] = {
-                "correct": 0,
-                "total": 0,
-                "stake": 0,
-                "payout": 0,
-            }
-
-        if predictoor_key not in stats["predictoor"]:
-            stats["predictoor"][predictoor_key] = {
-                "correct": 0,
-                "total": 0,
-                "stake": 0,
-                "payout": 0,
-                "details": set(),
-            }
-
-        if is_correct:
-            correct_predictions += 1
-            stats["pair_timeframe"][pair_timeframe_key]["correct"] += 1
-            stats["predictoor"][predictoor_key]["correct"] += 1
-
-        stats["pair_timeframe"][pair_timeframe_key]["total"] += 1
-        stats["pair_timeframe"][pair_timeframe_key]["stake"] += prediction.stake
-        stats["pair_timeframe"][pair_timeframe_key]["payout"] += prediction.payout
-
-        stats["predictoor"][predictoor_key]["total"] += 1
-        stats["predictoor"][predictoor_key]["stake"] += prediction.stake
-        stats["predictoor"][predictoor_key]["payout"] += prediction.payout
-        stats["predictoor"][predictoor_key]["details"].add(
-            (prediction.pair, prediction.timeframe, source)
-        )
-
-    print(f"Overall Accuracy: {correct_predictions/total_predictions*100:.2f}%")
-
-    for key, data in stats["pair_timeframe"].items():
-        pair, timeframe = key
-        accuracy = data["correct"] / data["total"] * 100
-        print(f"Accuracy for Pair: {pair}, Timeframe: {timeframe}: {accuracy:.2f}%")
-        print(f"Total stake: {data['stake']}")
-        print(f"Total payout: {data['payout']}")
-        print(f"Number of predictions: {data['total']}\n")
-
-    for predictoor, data in stats["predictoor"].items():
-        accuracy = data["correct"] / data["total"] * 100
-        print(f"Accuracy for Predictoor Address: {predictoor}: {accuracy:.2f}%")
-        print(f"Stake: {data['stake']}")
-        print(f"Payout: {data['payout']}")
-        print(f"Number of predictions: {data['total']}")
-        print("Details of Predictions:")
-        for detail in data["details"]:
-            print(f"Pair: {detail[0]}, Timeframe: {detail[1]}, Source: {detail[2]}")
-        print("\n")
-
+from pdr_backend.util.csvs import write_csv
```

Comment:
This file is in way better shape than before. What I was hoping for. Great stuff!